plugins/lsp/language-servers/nil-ls-settings.nix

```diff
@@ -0,0 +1,58 @@
+{ lib, helpers }:
+# Options:
+#  - https://github.com/oxalica/nil/blob/main/docs/configuration.md?plain=1#
+with lib;
+{
+  formatting = {
+    command = helpers.defaultNullOpts.mkNullable (types.listOf types.str) null ''
```

Comment:
I was under the impression `mkNullable` needed the plugin-default to be a string (`"null"`), but it looks like it gets passed through `toString`.

I guess we only need to pass in a string when we want to enforce formatting or display nested elements. 🤔

Reply:
Hmmm, we should be careful here.
Indeed, `toString null` is `""` and not `"null"`.

Reply:
Off-topic, but maybe `mkNullable` should use a (bespoke or third-party) pretty-stringify instead of `builtins.toString` to allow us to use actual `nix` and strings interchangeably?

For now, @khaneliman, could you switch to passing string defaults to `mkNullable` matching how it's done elsewhere?

Reply:
> Off-topic, but maybe `mkNullable` should use a (bespoke or third-party) pretty-stringify instead of `builtins.toString` to allow us to use actual `nix` and strings interchangeably?

I agree, this could be nice !

Reply:
Sounds good, updated to use the "null" string instead of null type.